packages/store/addon/-private/system/core-store.ts

```diff
@@ -843,8 +843,7 @@ abstract class CoreStore extends Service {
     has already been assigned an id, you can find the record again using just the lid.
 
     ```app/routes/post.js
-    let identifier = recordIdentifierFor(person);
-    store.findRecord({ lid: identifier.lid });
+    store.findRecord(identifier);
```

Comment:
```suggestion
    store.findRecord({ lid });
```

to make it clear it doesn't need type+id necessarily (which are already in example 2).
we could add a note that this also works by passing an identifier previously obtained from another ember-data api ala

```
Any identifier object returned by another ember-data API is able to be passed to this API.
```

or some such.